public/css/main.css

```diff
@@ -131,34 +131,35 @@ footer {
     max-width: 100%;
     margin-left: auto;
     margin-right: auto;
-    max-height: 30px;
 }
 
 .navbar-nav > li > a.navbarBtn {
-    background-color:#ffffff;
-    -moz-border-radius:15px;
-    -webkit-border-radius:15px;
-    border-radius:10px;
-    border:1px solid #2D2A3F;
-    display:inline-block;
+    background-color: #ffffff !important;
```

Comment:
What CSS are you overriding by using the `!important` tag here? Using this is generally bad practice. Is it possible to just modify the appropriate CSS instead of using the `!important` tag?

Reply:
Got it. I was just overriding some bootstrap CSS, so I'll just adjust the styling rule a little